Sources/vtk.js

```diff
@@ -1,3 +1,7 @@
+import globalThisShim from 'globalthis';
+
+const vtkGlobal = globalThisShim(); // returns native globalThis if compliant
```

Comment:
I would rather use the one from macro once exported. I don't want to spread our dependency to `globalthis` especially if we replace it down the road.

Reply:
Ok